sda_orchestrator/config/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""SDA orchestrator configuration.
+
+Configuration required for DOIs, REMS and other metadata.
+"""
+
+from os import environ
+from pathlib import Path
+from typing import Dict
+import json
+
+
+def parse_config_file(config_file: str) -> Dict:
+    """Load JSON schemas."""
+    file_path = Path(config_file)
+
```

Comment:
```suggestion
from sys import exit

def parse_config_file(config_file: str) -> Dict:
    """Load JSON schemas."""
    file_path = Path(config_file)
    if not file_path.is_file():
        exit(f"File {file_path} not found")
```
Check that file exists before opening it, so that it exits gracefully with a description and doesn't crash with stack trace.

Reply:
yes that is a good suggestion, but not sure I should exit ..., maybe a raise and logging the error